doc/api/process.md

```diff
@@ -1775,6 +1784,317 @@ Will generate an object similar to:
   tz: '2016b' }
 ```
 
+## Performance Timing API
+<!-- YAML
+added: REPLACEME
+-->
+
+The Performance Timing API provides an implementation of the
+[W3C Performance Timeline][] specification. The purpose of the API
+is to support collection of high resolution performance metrics.
+This is the same Performance API as implemented in modern Web browsers.
+
+```js
+const perf = process.performance;
+perf.mark('A');
+doSomeLongRunningProcess(() => {
+  perf.mark('B');
+  perf.measure('A to B', 'A', 'B');
+  const measure = perf.getEntriesByName('A to B')[0];
+  console.log(measure.duration);
+  // Prints the number of milliseconds between Mark 'A' and Mark 'B'
+});
+```
+
+### Class: Performance
```

Comment:
[High Resolution Time](https://www.w3.org/TR/hr-time-2/#the-performance-interface) also defines a `toJSON()` method.